js/script.js

```diff
@@ -0,0 +1,98 @@
+
+import {
+    $createButton,
+    $taskContainer
+} from './element.js'
+
+let isFormOpen = false
+let tasks = []
+
+const getUniqueId = () => Date.now();
+
+const makeForm = (taskInfo = {}) => {
+    let $form = document.createElement('li')
+    let $title = document.createElement('input');
+    let $addButton = document.createElement('button')
+
+    $addButton.innerHTML = "Add"
+    $title.setAttribute('name', 'title');
+    $title.setAttribute('placeholder', "Enter your task name")
+    $title.setAttribute('id', 'title')
+    $addButton.setAttribute('id', 'add-task')
+    $addButton.addEventListener('click', () => addTask())
+
+    if (taskInfo.title) {
+        $title.value = taskInfo.title
+    }
+
+    $form.appendChild($title);
+    $form.appendChild($addButton)
+
+    return $form
+}
+
+const generateTask = (task) => {
+    let $listElement = document.createElement('li');
+    $listElement.setAttribute('id', task.id);
+    $listElement.innerText = task.title;
+    return $listElement;
+}
+
+const renderTasks = (taskContiner, tasks) => {
+    tasks.map((task) => {
+        let element = generateTask(task)
+        taskContiner.appendChild(element)
+    })
+}
+
+const addTask = () => {
+    let $title = document.getElementById('title')
+    let titleValue = $title?.value?.trim();
```

Comment:
the input value is not santized

Reply:
You can declare this in the **element.js** file and export it from there, as I believe it is the common input file for task title. Also please change the name.

Reply:
@ich-wtag Bhaiya, I declared the input element locally because I made a dynamic form. When a form is created, this element exists otherwise, it doesn't exist. In this case, is it the right way to define its global scope?. or make a static form for it?